AppDB/appscale/datastore/backup/backup_recovery_helper.py

```diff
@@ -20,12 +20,12 @@
 from appscale.datastore.backup.br_constants import BACKUP_ROLLBACK_SUFFIX
 from appscale.datastore.backup.br_constants import PADDING_PERCENTAGE
 from appscale.datastore.backup.br_constants import StorageTypes
-from appscale.datastore.unpackaged import APPSCALE_LIB_DIR
 
-sys.path.append(APPSCALE_LIB_DIR)
-import appscale_info
-from constants import APPSCALE_DATA_DIR
+from appscale.common import appscale_info
```

Comment:
Could you put these in alphabetical order above the appscale.datastore block of imports?